runtime/browser/xwalk_utils.cc

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/browser/xwalk_utils.h"
+
+namespace xwalk_utils {
+
+bool IsLocalHost(const GURL& url) {
+  std::string host = url.host();
+  return host.compare("localhost") ||
```

Comment:
think it should rather be `host.compare("localhost") == 0` and same for others.. http://www.cplusplus.com/reference/string/string/compare/ (looks like browser tests failure on Linux might be related)